README.md

```diff
@@ -0,0 +1,28 @@
+# libavrlite: a thin abstraction library for the AVR platform
```

Comment:
I'll have to make some edits to the README, but they're hard to convey so just keep this for now and I'll update it later.